composer.json

```diff
@@ -12,10 +12,11 @@
         "bin/daenerys"
     ],
     "require": {
+        "php": ">7.0.0",
```

Comment:
You're right.
Am 07.06.2016 22:47 schrieb "Austen McDonald" <notifications@github.com>:

> In composer.json
> <https://github.com/lotgd/core/pull/37#discussion_r66149494>:
>
> > @@ -12,10 +12,11 @@
> >          "bin/daenerys"
> >      ],
> >      "require": {
> > +        "php": ">7.0.0",
>
> Why not ^7.0.0?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lotgd/core/pull/37/files/ac16c54591b7ef1c0b036074cbc7de49efad1da4#r66149494>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABULgjMhmPr-EwLVXL6_8_0kOf-mP195ks5qJdjMgaJpZM4IwERw>
> .
>